### ISSUE TITLE
[Privacy policy] Links are not focusable

### ISSUE THREAD

**Issue by svinkle** [CovidShield/mobile#40]
## Issue summary

When using swipe/touch gestures to navigate through the Privacy Policy, content links do not receive focus. Link text is included in the paragraph text with no notice of their existence. This makes accessing content linked from this view impossible for screen reader users and voice dictation users, leading to a frustrating user experience.

![Screen Shot 2020-05-25 at 4 57 57 PM](https://user-images.githubusercontent.com/1392632/82842806-eca2ad80-9ea8-11ea-83c9-5509f4792ea6.png)

#### JS

```js
// react-native-markdown-display/renderRules.js

link: (node, children, parent, styles, onLinkPress) => (
  <Text
    key={node.key}
    style={styles.link}
    onPress={() => openUrl(node.attributes.href, onLinkPress)}
  >
    {children}
  </Text>
);
```

## Steps to reproduce

1. Load the app into the iOS or Android simulator
2. Expand the Sheet content
3. Activate the "Read privacy policy" control
3. Enable [VoiceOver screen reader](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) on macOS
4. Use the Virtual Cursor to navigate to the view content

### Expected

- Links to be focusable, announce its label, and conveyed as a "link".

### Actual

- Links are not focusable.

## Recommendation

Since this is a third-party lib, making changes in a timely manner may be difficult. Try to find an alternate markdown renderer, or output the content in a `Text` component, using `Button` components with `accessibleRole="link"` props to output focusable links.

## Specifications

- **Component:** CovidShield [Privacy policy]
- **WCAG Principle:** Operable
- **WCAG SC:** 2.1.1 Keyboard
- **Severity:** High
- **Effort:** Unknown
